testsuite/MDAnalysisTests/analysis/test_psa.py

```diff
@@ -126,10 +125,12 @@ def test_get_coord_axes_bad_dims(self):
             PSA.get_coord_axes(np.zeros((5,5,5,5)))
 
     def test_dist_mat_to_vec_func_out_of_bounds(self):
-        """Test that ValueError is raised when i or j is
+        """Test that ValueError is raised when i or j or both are
         out of bounds of N"""
 
         with self.assertRaises(ValueError):
+            PSA.dist_mat_to_vec(5, 6, 4)
```

Comment:
Add a comment to tell what index is out of bonds. It is obvious when you are in the context of writing the test, but it makes you loose quite a lot of time when you have to figure it out latter on.